Arrowgene.Ddon.GameServer/Characters/JobManager.cs

```diff
@@ -233,6 +233,25 @@ public void UnlockSkill(IDatabase database, GameClient client, CharacterCommon c
                     SkillId = skillId,
                     SkillLv = skillLv
                 });
+
+                //Find on the palletes (if any) where the skill is set and notify party. Can occur at two locations (Main + Secondary) for players.
+                List<CustomSkill?> equippedCustomSkillList = character.EquippedCustomSkillsDictionary[job];
+                var slotIndices = Enumerable.Range(0, equippedCustomSkillList.Count)
+                    .Where(i => equippedCustomSkillList[i] != null && equippedCustomSkillList[i].SkillId == skillId)
+                    .ToList();
+                foreach (int slotIndex in slotIndices)
+                {
+                    client.Party.SendToAll(new S2CSkillCustomSkillSetNtc()
```

Comment:
When a skill is equipped the entire server is notified, not just the party. I don't remember the reasoning exactly but I believe it was so the change was reflected in the arisen profile. Should we do the same here? 🤔

Reply:
I was mistaken, my bad, i thought we were sending it to the entire server. It's probably fine either way both in functionality and performance wise.

The crashes in Arisen profile aren't related to this, it's definitely uninitialized memory, where the client expects some lists in Arisen profile to have an amount of items while the lists we're sending are empty. This is why when it doesn't crash you see seemingly random values, they're whatever was in memory in those addresses